.gitignore

```diff
@@ -174,4 +174,6 @@ catboost_info
 generated
 # npm
 node_modules/
-mlruns
\ No newline at end of file
+mlruns
+example.ipynb
+logged_demo.ipynb
```

Comment:
I wouldn't mention personal notebooks to gitibnore, if it help, you can use `git add -u .` to add only "tracked" files.